app/uk/gov/hmrc/softdrinksindustrylevy/connectors/DesConnector.scala

```diff
@@ -153,6 +158,11 @@ class DesConnector(
     }
   }
 
+  def displayDirectDebit(sdilRef: String)(implicit hc: HeaderCarrier): Future[DisplayDirectDebitResponse] = {
+    val uri = s"$desDirectDebitUrl/cross-regime/direct-debits/zsdl/zsdl/$sdilRef"
```

Comment:
does it matter if zsdl is uppercase as in other endpoints or not?

Reply:
also I suppose /zsdl/zsdl is not a typing mistake and that's the endpoint in DES?

Reply:
I believe the capitalisation is important for this API since the latest version of it. The validation is now performed at ETMP rather than DES, and ETMP has been sensitive around the case of these names for other APIs. We've used this API ok so far in E2E tests for the direct debit service with lowercase names.

Yes, it definitely looks strange, but "zsdl" is both the name of the tax regime and the name of the tax identifier!